commitizen-tools/commitizen
# cli(script): gen screenshot for cli helps

## Description

Add a script to generate screenshot of all cli help contents. 

## Checklist



## Expected behavior

Execute `scripts/gen_cli_help_screenshots.py` will generate screenshots of cli help instructions

## Steps to Test This Pull Request

1. Execute `scripts/gen_cli_help_screenshots.py`
2. Check each of content insides `commitizen.cli`'s `data` is match its screenshot in `docs/images/cli_help/` folder.


## Additional context

Considering the purpose of the file is more closed to a script, put it into scripts folder.
Should add test and update the documentation accordinately after the change is accepted.
